packages/profiling-node/src/utils.ts

```diff
@@ -213,6 +215,10 @@ function createProfileChunkPayload(
 
   const profile: ProfileChunk = {
     chunk_id: chunk_id,
+    client_sdk: {
+      name: sdk?.name ?? 'unknown',
```

Comment:
Can we set it to some fallback that indicates the platform? Maybe something like `sentry.javascript.node` or whatever the default is?

Reply:
Swapped the default for sdk name, but I'm not sure what to do with version. We could pass an empty string if it'll pass validation

Reply:
> why are these typed as optional btw? Wouldnt we expect these to always be present

because it's a relic of old sdk design and we loath to make these kind of breaking changes.

SDK name should pretty much be always there, only 0.005% edge case is custom clients.

> Swapped the default for sdk name, but I'm not sure what to do with version. We could pass an empty string if it'll pass validation

We shouldn't set a version - or null it out to `0.0.0`.

Reply:
> because it's a relic of old sdk design and we loath to make these kind of breaking changes.

I suspected something like this :) Seems like the change in relay expects version to be set, so I just set it to 0.0.0 as a default